classes/02class/exercises/c02-network09/drii-cavalcanti/c02-network09.txt

```diff
@@ -0,0 +1,17 @@
+Security Groups -> Create security group
+Security group name*: public-sg | private-sg
```

Comment:
Is it the 2 security groups creation in the same line split by `|`?

Can you break it down and paste the Inbound rules for them?